.github/workflows/ExportPluto.yaml

```diff
@@ -0,0 +1,53 @@
+---
+name: Export Pluto notebooks
+on:
+    push:
+        branches:
+            - main
+            - master
```

Comment:
We could add this branch here in case we don't want to try it directly on `master`:
```yaml
on:
    push:
        branches:
            - main
            - master
            - pluto-static-export
```